charts/ChartConfig.tsx

```diff
@@ -172,11 +181,11 @@ export class ChartConfigProps {
     @observable.ref owidDataset?: OwidDataset = undefined
 
     @observable.ref selectedData: EntitySelection[] = []
-    @observable.ref minTime?: number = undefined
-    @observable.ref maxTime?: number = undefined
+    @observable.ref minTime?: TimeBound = undefined
+    @observable.ref maxTime?: TimeBound = undefined
 
-    @observable.ref timelineMinTime?: number = undefined
-    @observable.ref timelineMaxTime?: number = undefined
+    @observable.ref timelineMinTime?: Time = undefined
```

Comment:
Do we need a minTime and a timelineMinTime? Do we have 2 time filters/

Reply:
Not great naming... but one of them sets the start & end handles in the timeline, and the other limits what the minimum/maximum possible value can be selected (so you can hide some years from the timeline)

Reply:
I think it's good to start writing some docstrings around these properties, since they are pretty stable given they are in the database, for better or worse...